corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -473,6 +482,32 @@
         {% endcompress %}
         {% endif %}
 
+
+        {% if request.use_maps or request.use_maps_async %}
+        {% compress js %}
+        <script type="text/javascript" src="{% static 'jquery-color/jquery.color.js' %}"></script>
+        <script type="text/javascript" src="{% static 'leaflet/dist/leaflet.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if reqeust.maps_prefer_canvas %}
+        <script>L_PREFER_CANVAS = true;</script>
+        {% endif %}
+
+        {% if request.use_maps %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if request.use_maps_async %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/async_maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
```

Comment:
@snopoke I don't believe that this PR is the cause of those issues. I actually think it's the fix. Look at the timestamp of the issue vs when this PR got merged in. I believe this isn't in master yet? The ticket seems like it's talking about the CDN issues that this PR addresses.